modules/apps/object/object-test/src/testFunctional/tests/ObjectClientExtensions.testcase

```diff
@@ -0,0 +1,111 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	static var clientExtensionSamples = "";
+	static var companyWebId = "";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		var companyWebId = JSONCompany.getWebId();
+	}
+
+	tearDown {
+		if (isSet(clientExtensionSamples)) {
+			for (var clientExtension : list ${clientExtensionSamples}) {
+				AntCommands.runCommand("build-test-liferay-sample-workspace.xml", "undeploy-workspace-client-extension -Dvirtual.instance.id=${companyWebId} -Dworkspace.client.extension.name=${clientExtension}");
+			}
+
+			var clientExtensionSamples = "";
+		}
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-179367 - Verify CRUD functionality with a created custom object definition using a client extension as the storage type."
+	@priority = 5
+	test AssertCRUDWithClientExtensionAsStorageType {
+		property custom.properties = "feature.flag.LPS-135430=true";
+		property portal.acceptance = "runJhosseph";
```

Comment:
Please remove this property and let's monitor it for stability first before adding it to acceptance